test/test.ts

```diff
@@ -462,50 +505,54 @@ Deno.test("sqlite", async (t) => {
     const [result] = db
       .prepare("select deno_add(?, ?)")
       .enableCallback()
-      .value<[number]>(1, 2)!;
+      .value<[number]>([1, 2])!;
     assertEquals(result, 3);
 
     const [result2] = db
       .prepare("select deno_uppercase(?)")
       .enableCallback()
-      .value<[string]>("hello")!;
+      .value<[string]>(["hello"])!;
     assertEquals(result2, "HELLO");
 
     const [result3] = db
       .prepare("select deno_buffer_add_1(?)")
       .enableCallback()
-      .value<[Uint8Array]>(new Uint8Array([1, 2, 3]))!;
+      .value<[Uint8Array]>([new Uint8Array([1, 2, 3])])!;
     assertEquals(result3, new Uint8Array([2, 3, 4]));
 
-    const [result4] = db.prepare("select deno_add(?, ?)").value<[number]>(
+    const [result4] = db.prepare("select deno_add(?, ?)").value<[number]>([
       1.5,
       1.5,
-    )!;
+    ])!;
     assertEquals(result4, 3);
 
     const [result5] = db
       .prepare("select regexp(?, ?)")
       .enableCallback()
-      .value<[number]>("hello", "h.*")!;
+      .value<[number]>(["hello", "h.*"])!;
     assertEquals(result5, 1);
 
     const [result6] = db
       .prepare("select regexp(?, ?)")
       .enableCallback()
-      .value<[number]>("hello", "x.*")!;
+      .value<[number]>(["hello", "x.*"])!;
     assertEquals(result6, 0);
 
     db.exec("create table aggr_test (value integer)");
     db.exec("insert into aggr_test (value) values (1)");
     db.exec("insert into aggr_test (value) values (2)");
     db.exec("insert into aggr_test (value) values (3)");
 
-    const stmt = db.prepare("select deno_sum_2x(value) from aggr_test");
-    stmt.callback = true;
-    const [result7] = stmt.value<[number]>()!;
-    assertEquals(result7, 12);
-    // Releases lock from table.
-    stmt.finalize();
+    // TODO figure out why the following error results from this test:
```

Comment:
@DjDeveloperr Due to changes to FFI, the aggregate test is breaking. After some research, it is due to bigint and number, but I am not completely sure where the error originates. Would you have an idea?

Reply:
There are a lot of changes in FFI which would cause type errors mostly - I will make a separate PR for now that takes care of all that